cloudfoundry/resource_cf_service_key.go

```diff
@@ -146,7 +146,7 @@ func resourceServiceKeyCreate(ctx context.Context, d *schema.ResourceData, meta
 
 		// Last operation initial or inprogress or job not completed, continue polling
 		return false, nil
-	}, 5*time.Second, 60*time.Second)
+	}, 5*time.Second, 300*time.Second)
```

Comment:
Instead of just modifying this value, I wonder if it's more feasible to integrate the the Timeouts setting of resources as in https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts, so that the default can stay at 60 seconds, but can be configured with something like

```
resource "cloudfoundry_service_key" "example" {
  # ...

  timeouts {
    create = "300s"
  }
}
```